src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -97,9 +98,6 @@
       <Link>Common\System\Net\DebugSafeHandleMinusOneIsInvalid.cs</Link>
     </Compile>
     <!-- System.Net common -->
-    <Compile Include="$(CommonPath)\System\Net\ContextAwareResult.cs">
```

Comment:
I see this being removed, but ContextAwareResult.cs and friends aren't being deleted.  Are they still being used by other System.Net projects?